client/src/common/components/Members/components/InviteNewUser.jsx

```diff
@@ -6,6 +6,7 @@ import PendingButton from 'common/components/PendingButton';
 import { PreloaderTheme } from 'common/components/Preloader';
 import { PROJECT_NAME } from 'common/constants/variables';
 import { IntlPropType } from 'common/constants/propTypes';
+import './InviteNewUser.scss';
```

Comment:
Please split external and internal import groups.

Reply:
Here imports are split. `import { IntlPropType } from 'common/constants/propTypes';` is our internal import